mojo/edk/embedder/scoped_platform_handle.h

```diff
@@ -54,6 +52,8 @@ class ScopedPlatformHandle {
 
  private:
   PlatformHandle handle_;
+
+  MOJO_MOVE_ONLY_TYPE(ScopedPlatformHandle);
```

Comment:
the definition of this macro changed in the mojo SDK, so in order to get the updated macro you should also update the mojo_sdk entry in //DEPS